src/content/docs/fr/guides/server-islands.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Îlots de serveur
+description: Combinez du HTML statique très performant avec du contenu dynamique rendu par le serveur.
+i18nReady: true
+---
+
+Les îlots de serveurs vous permettent de rendre à la demande des « îles » dynamiques ou personnalisées individuellement, sans sacrifier les performances du reste de la page.
+
+Cela signifie que votre visiteur verra les parties les plus importantes de votre page plus tôt, et permet à votre contenu principal d'être mis en cache de manière plus intensive, ce qui permet d'obtenir des performances plus rapides.
+
+## Composants d'île de serveur
+
+Un îlot de serveur est un [composant Astro](/fr/basics/astro-components/) normal rendu par le serveur, qui est chargé de différer le rendu jusqu'à ce que son contenu soit disponible.
+
+Votre page sera rendue immédiatement avec tout [contenu de secours](#contenu-de-secours-de-l%C3%AEle-du-serveur). Ensuite, le contenu du composant est récupéré sur le client et affiché lorsqu'il est disponible.
+
+Avec [un adaptateur installé](/fr/guides/on-demand-rendering/#server-adapters) pour effectuer le rendu différé, ajoutez la directive [`server:defer`](/fr/reference/directives-reference/#directives-serveur) à n'importe quel composant de votre page pour le transformer en sa propre île :
+
+```astro title="src/pages/index.astro" "server:defer"
+---
+import Avatar from '../components/Avatar.astro';
+---
+<Avatar server:defer />
+```
+
+Ces composants peuvent faire [tout ce que vous feriez normalement dans une page rendue à la demande](/fr/guides/on-demand-rendering/#on-demand-rendering-features) à l'aide d'un adaptateur, comme récupérer du contenu et accéder aux cookies :
+
+```astro title="src/components/Avatar.astro"
+---
+import { getUserAvatar } from '../sessions';
+const userSession = Astro.cookies.get('session');
+const avatarURL = await getUserAvatar(userSession);
+---
+<img alt="Avatar de l'utilisateur" src={avatarURL} />
+```
+
+## Contenu de secours de l'île du serveur
+
+Lorsque vous utilisez l'attribut `server:defer` sur un composant pour retarder son rendu, vous pouvez "slot" un contenu de chargement par défaut en utilisant le slot nommé `"fallback"` inclus.
```

Comment:
I think we can translate this slot since it's the verb and not the built-in slot feature:
```suggestion
Lorsque vous utilisez l'attribut `server:defer` sur un composant pour retarder son rendu, vous pouvez « insérer » un contenu de chargement par défaut en utilisant le slot nommé `"fallback"` inclus.
```

Reply:
Oh I hesitated for this one to edit. Okay good!